[2023-10-07 08:22:20,541] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart.update_d_customer scheduled__2023-10-01T08:17:37.691037+00:00 [queued]>
[2023-10-07 08:22:20,549] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart.update_d_customer scheduled__2023-10-01T08:17:37.691037+00:00 [queued]>
[2023-10-07 08:22:20,549] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-07 08:22:20,550] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-07 08:22:20,551] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-07 08:22:20,559] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): update_d_customer> on 2023-10-01 08:17:37.691037+00:00
[2023-10-07 08:22:20,562] {standard_task_runner.py:52} INFO - Started process 1652 to run task
[2023-10-07 08:22:20,564] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'sales_mart', 'update_d_customer', 'scheduled__2023-10-01T08:17:37.691037+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/sprint3.py', '--cfg-path', '/tmp/tmpqswcvwuk', '--error-file', '/tmp/tmp1uan8dp8']
[2023-10-07 08:22:20,565] {standard_task_runner.py:80} INFO - Job 41: Subtask update_d_customer
[2023-10-07 08:22:20,606] {task_command.py:369} INFO - Running <TaskInstance: sales_mart.update_d_customer scheduled__2023-10-01T08:17:37.691037+00:00 [running]> on host 36d9c57a9c84
[2023-10-07 08:22:20,651] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=student@example.com
AIRFLOW_CTX_DAG_OWNER=student
AIRFLOW_CTX_DAG_ID=sales_mart
AIRFLOW_CTX_TASK_ID=update_d_customer
AIRFLOW_CTX_EXECUTION_DATE=2023-10-01T08:17:37.691037+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-01T08:17:37.691037+00:00
[2023-10-07 08:22:20,657] {base.py:68} INFO - Using connection ID 'postgresql_de' for task execution.
[2023-10-07 08:22:20,662] {sql.py:315} INFO - Running statement: insert into mart.d_customer (customer_id, first_name, last_name, city_id)
select customer_id, first_name, last_name, max(city_id) from staging.user_order_log
where customer_id not in (select customer_id from mart.d_customer)
group by customer_id, first_name, last_name, parameters: None
[2023-10-07 08:22:20,664] {sql.py:324} INFO - Rows affected: 182
[2023-10-07 08:22:20,675] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=sales_mart, task_id=update_d_customer, execution_date=20231001T081737, start_date=20231007T082220, end_date=20231007T082220
[2023-10-07 08:22:20,697] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-07 08:22:20,720] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
